platform/mds/mds/src/main/java/org/motechproject/mds/osgi/MdsBundleWatcher.java

```diff
@@ -296,6 +300,26 @@ private void processAnnotationScanningResults(List<EntityProcessorOutput> entity
         }
     }
 
+    private void updateUiChangedFields(Collection<FieldDto> fieldsToUpdate, String entityClassName) {
```

Comment:
Doesn't this belong in the field/uifilterable processor? This code seems out of place here